docs/source/index.rst

```diff
@@ -1,11 +1,60 @@
 ipywidgets
 ==========
 
-Full Table of Contents
-----------------------
+.. only:: html
+
+    :Release: |release|
+    :Date: |today|
+
+**ipywidgets**, also known as jupyter-widgets or simply widgets, are
+`interactive HTML widgets <https://github.com/jupyter-widgets/ipywidgets/blob/master/docs/source/examples/Index.ipynb>`_
```

Comment:
I found it a bit jarring to be linked out github here. Though I'm not sure where (if anywhere) this link would go otherwise. It's just that github is one of the few places where there are no actually interactive html widgets for a reader to play with.

Maybe a binder link would be best?

Reply:
Sure. That's reasonable. This PR was simply to restructure the content structure and less focused on the content itself. I think changes like these would be great follow-up PRs.